plugins/google-sheets/src/App.tsx

```diff
@@ -87,9 +87,17 @@ export function AuthenticatedApp({ pluginContext, setContext }: AuthenticatedApp
     }, [isUserInfoError, isSelectSheetError, setContext])
 
     useLayoutEffect(() => {
+        const width = sheetTitle ? 360 : 320
+        const height = sheetTitle ? 425 : 345
```

Comment:
No `!== null` on purpose? If not, I'd compare to `null` explicitly in `resizable` too.